io.qt.qtwebengine.BaseApp.json

```diff
@@ -0,0 +1,87 @@
+{
+    "id" : "io.qt.qtwebengine.BaseApp",
+    "branch": "5.13",
+    "runtime" : "org.kde.Platform",
+    "sdk" : "org.kde.Sdk",
+    "runtime-version" : "5.13",
+    "modules" : [
+        {
+            "name" : "qt5-qtwebengine",
+            "buildsystem" : "qmake",
+            "cleanup-platform" : [
+                "/bin",
+                "/mkspecs"
+            ],
+            "make-install-args": ["INSTALL_ROOT=/app/potatoes"],
+            "config-opts" : [
+                "--",
+                "-system-ffmpeg",
+                "-webengine-proprietary-codecs",
+                "-webengine-icu",
+                "-webp"
+            ],
+            "post-install" : [
+                "cp -r /app/potatoes/usr/* /app",
+                "rm -rf /app/potatoes"
+            ],
+            "modules" : [
+                "shared-modules/python2.7/python-2.7.json",
```

Comment:
If it's needed only build time, copy-paste entire module and add `"cleanup": ["*"]`